code/src/itest/plugin/lsttokens/editcontext/template/FaceIntegrationTest.java

```diff
@@ -52,9 +52,9 @@ public void setUp() throws PersistenceLayerException, URISyntaxException
 		super.setUp();
 		TokenRegistration.register(m);
 		primaryContext.getVariableContext().assertLegalVariableID(
-			primaryContext.getActiveScope(), opManager, "Face");
+			"Face", primaryContext.getActiveScope(), opManager);
```

Comment:
Consider having Strings like "Face" as a final static somewhere. It's just error prone to typos and makes it harder if it needs to change for whatever reason later.